editor/script/script_editor_plugin.cpp

```diff
@@ -4378,6 +4386,9 @@ ScriptEditor::ScriptEditor(WindowWrapper *p_wrapper) {
 	script_name_label->set_text_overrun_behavior(TextServer::OVERRUN_TRIM_ELLIPSIS);
 	script_name_label->set_focus_mode(FOCUS_ACCESSIBILITY);
 	script_name_label->set_h_size_flags(SIZE_EXPAND_FILL);
+	script_name_label->set_mouse_filter(MOUSE_FILTER_STOP);
+	script_name_label->set_tooltip_text(TTRC("Double click to navigate to the script."));
```

Comment:
This is potentially confusing, the user could think it will navigate to it in the FileSystem dock. So I'd say it should somehow mention the script list.